Makefile

```diff
@@ -591,7 +577,7 @@ ui-component-tests:
 	make -C ui test-component
 
 .PHONY: test
-test: go-unit-tests ui-test shell-unit-tests
+test: go-postgres-unit-tests ui-test shell-unit-tests
```

Comment:
Would it make sense to rename this to `go-unit-tests`?

Reply:
I thought about it but this explicitly mentions the postgres requirement

Reply:
fair enough.